POLYFILLS.md

```diff
@@ -6,20 +6,50 @@ browsers yet. In order to maximize browser compatibility, you should install
 [polyfills](https://en.wikipedia.org/wiki/Polyfill_(programming)) to fill in the
 gaps for some of the newest features.
 
+---
+
+ - ✅ Natively supported
+ - 🚧 Available with polyfill
+ - 🚫 Not available
+ - 🎌 Behind a flag, unstable
+
+Feature                   | Chrome | Canary | Safari 12 | Firefox 64 | Firefox 62 | Edge  | IE 11
+--------------------------|--------|--------|-----------|------------|------------|-------|------
+Resize Observer¹          |     ✅ |     ✅ |        🚧 |         🚧 |         🚧 |    🚧 |   🚧
```

Comment:
Maybe an offshoot of the discussions between you and @yuinchien, but maybe we can use a softer "polyfill required" icon, so it doesn't look like it's only supported in Chrome

Reply:
I actually should have replaced these with some Material Design icons she recommended. I'll fix them up before merge.

Reply:
After some discussion, filed #405 and @yuinchien will be taking on that work herself 🙇